test/multi-ns/main_test.go

```diff
@@ -0,0 +1,116 @@
+package multi_ns_test
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+var (
+	//flags used
+	kubeconfig  *string
+	clusterName = flag.String("clusterName", "test", "Name of kind cluster for this test")
+	ipAddr      = flag.String("remoteIp", "127.0.0.1", "Set ENV to REMOTE_IP")
+	nsmPath     = flag.String("nsmPath", "../../scripts/vl3/nsm_install_interdomain.sh", "Path of script to install NSM")
+	nsePath     = flag.String("nsePath", "../../scripts/vl3/vl3_interdomain.sh", "Path of script to install NSE")
+
+	clientset *kubernetes.Clientset
+)
+
+func TestMain(m *testing.M) {
+	//Get kubeconfig
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.Parse()
+	//Remove existing cluster which has the same name
+	fmt.Println("Removing old cluster...")
+	err := removeExistingKindCluster(*clusterName)
+	if err != nil {
+		fmt.Printf("Failed to remove old cluster '%s' with err: %v", *clusterName, err.Error())
+	}
```

Comment:
In case the clusters were not removed because of some error, we would want this test to continue?

Reply:
Ah, my mistake. Will add os.Exit() after the error message.